<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleEdit</class>
 <widget class="QMainWindow" name="ParticleEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Editor</string>
  </property>
  <widget class="QToolWindowManager" name="centralwidget">
   <widget class="PropertyEditor" name="treeView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="Viewport" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>200</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
    <property name="source">
     <url>
      <string>qrc:/QML/qml/Emitters.qml</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuEmitter">
     <property name="title">
      <string>Emitter</string>
     </property>
     <addaction name="actionEmitter"/>
     <addaction name="actionBeam"/>
     <addaction name="actionMesh"/>
    </widget>
    <addaction name="menuEmitter"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_emitter"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionDelete_emitter">
   <property name="text">
    <string>Delete emiter</string>
   </property>
  </action>
  <action name="actionEmitter">
   <property name="text">
    <string>Create Emitter</string>
   </property>
  </action>
  <action name="actionMesh">
   <property name="text">
    <string>Create Mesh</string>
   </property>
  </action>
  <action name="actionBeam">
   <property name="text">
    <string>Create Beam</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Images/Pause.png</normaloff>:/Images/Pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Images/Repeat.png</normaloff>:/Images/Repeat.png</iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>PropertyEditor</class>
   <extends>QTreeView</extends>
   <header>editors/propertyedit/propertyeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QToolWindowManager</class>
   <extends>QWidget</extends>
   <header>managers/toolwindowmanager/qtoolwindowmanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Viewport</class>
   <extends>QWidget</extends>
   <header>graph/viewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
